clients/privacy-center/components/Card.tsx

```diff
@@ -57,13 +57,7 @@ const Card: React.FC<CardProps> = ({
     >
       {title}
     </Text>
-    <Text
-      color="gray.600"
-      fontSize="xs"
-      fontWeight="normal"
-      lineHeight="16px"
-      noOfLines={3}
-    >
+    <Text color="gray.600" fontSize="xs" fontWeight="normal" lineHeight="16px">
```

Comment:
Nit: Could you leave these on separate lines as it's much easier to see the diffs at that point

Reply:
I think Prettier forced it to go to one line.